crates/holochain/src/core/workflow/app_validation_workflow/network_call_tests.rs

```diff
@@ -610,3 +712,145 @@ async fn check_cascade(
         .unwrap();
     validation_package
 }
+
+#[tokio::test(threaded_scheduler)]
+#[ignore = "Only shows a problem, doesn't prove something is correct"]
+/// This test shows a slow read issue.
+/// The exact same code running here in this test is 10x
+/// faster then when it is run by the cell
+async fn slow_lmdb_reads_test() {
```

Comment:
Maybe add some context about how this may not actually be a problem, but just something to look out for?
```suggestion
#[ignore = "Only shows a potential problem, doesn't prove something is correct"]
/// This test shows a potential slow read issue.
/// The exact same code running here in this test is 10x
/// faster then when it is run by the cell
///
/// This may not turn out to be a real issue, but this illustrates a way to reproduce this behavior,
/// and may be something we want to investigate more in the future.
async fn slow_lmdb_reads_test() {
```